docs/how to/use-transferbench.rst

```diff
@@ -116,5 +140,53 @@ Broadcast from GPU 0 to GPU 0 and GPU 1::
 
    1 16 (G0->G0->G0G1)
 
+.. note::
+
+   Running TransferBench with no arguments displays usage instructions and detected topology information.
```

Comment:
@gilbertlee-amd Can we show how to run transferbench?

Reply:
Not quite sure what you mean.  You mean like "./TransferBench"?

Reply:
> Not quite sure what you mean. You mean like "./TransferBench"?

@gilbertlee-amd Yes, showing the command with arguments.

Reply:
@gilbertlee-amd The usage of arguments with ./TransferBench command is slightly complicated and confusing. I guess I would omit this for now as it would again require to explain the arguments and usage as per the use-case.